presto-native-execution/presto_cpp/main/types/tests/CMakeLists.txt

```diff
@@ -89,3 +89,24 @@ target_link_libraries(
   velox_tpch_connector
   gtest
   gtest_main)
+
+add_executable(presto_function_metadata_test FunctionMetadataTest.cpp)
+
+add_test(
+  NAME presto_function_metadata_test
+  COMMAND presto_function_metadata_test
+  WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR})
+
+target_link_libraries(
+  presto_function_metadata_test
+  gtest
```

Comment:
We probably want to reorder the libraries - first presto ones in alphabetical order followed by velox in alphabetical order and system libraries last in lphabetical order (see previous `presto_to_velox_connector_test` library).